packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx

```diff
@@ -0,0 +1,160 @@
+import { Button, Icon, Text } from 'blockchain-info-components'
+import { FlyoutWrapper } from 'components/Flyout'
+import { FormattedMessage } from 'react-intl'
+import { LinkDispatchPropsType, OwnProps } from '.'
+import React from 'react'
+import styled from 'styled-components'
+
+import {
+  DisplayTitle,
+  NumberContainer,
+  NumberDescription,
+  NumberWrapper,
+  RowNumber,
+  Subcontent,
+  SubTitle
+} from '../template.rejected.styles'
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+`
+
+const Title = styled(Text)`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 7px;
+`
+
+const ShortTitleContainer = styled.div`
+  max-width: 154px;
```

Comment:
Is this needed?

Reply:
According to design, they wanted to have it like that in 2 lines, we already have same label in use and I added this to have it into 2 lines for this step